### ISSUE TITLE
Void ore miner and void ressource miner not working

### ISSUE THREAD

**Issue by wanou067** [ValkyrieofNight/EnvironmentalTech#137]
I built the void ressource miner and the void ore miner but they does not work
when I build it even in creative mode or in survival the laser does not light up and the structure is unusable!
I specify that I am on a server forge version 1.10.2 latest with a password similar to Feed the Beast Beyond

**Comment by Chinthor** [comment]
I have the same issue. I'm running the latest version of the DireWolf20 pack. Wanting to fix this, I manually updated EvTech and ValkyLib to latest versions.

So far, I've tried EnderIO energy conduits and placing an RFTools energy cell directly on the void miner. with or without a redstone signal. Placed a chest directly on the miner or connected with an item duct and a servo. It still does not draw any power. 

Confirmed bedrock below, no laser animation shows up.

**Comment by ValkyrieofNight** [comment]
Give me pictures. If the laser isn't showing then you are missing a block.

**Comment by Chinthor** [comment]
You are right. Came back to retract issue, and you've already been at it. I was reading the requirements wrong. All is good!

**Comment by ValkyrieofNight** [comment]
Haha glad to hear. There will be better ways to figure out if you've missed a block soon also

**Comment by Chinthor** [comment]
With this thread open, I'll just throw this in. Even when it started working (drawing power, showing the laser, yay!) There was never a "structure complete" msg in the chat log. I checked backwards a bit. Also, when I hit it with the assembler, it still listed the top line (was either panels or structure blocks) in red. Is the "complete" msg coming back? Or could the text color of blocks that are accounted for be green/blue and missing ones red in the on-hit message? Just a thought. Now back to making a base made entirely of frick'n lasers! EvTech, and Draconic, and AcAdd, oh my!

**Comment by wanou067** [comment]
For me the problem persist, did someone cand help me ?

**Comment by wanou067** [comment]
http://image.noelshack.com/fichiers/2018/39/1/1537805008-2018-09-24-18-02-38.png
http://image.noelshack.com/fichiers/2018/39/1/1537805010-2018-09-24-18-02-49.png
http://image.noelshack.com/fichiers/2018/39/1/1537805017-2018-09-24-18-02-55.png

**Comment by Duergy** [comment]
ValkyrieLib Version:  valkyrielib-1.12.2-2.0.15.1.jar
Environmental Tech Version:  environmentaltech-1.12.2-2.0.15.1.jar
Description of bug: 
The Void Ore miner Multiblock dont get completed.
(If Applicable)Modpack Name: 
I create a new profile just with
- forge 14.23.4.2760
- valkyrielib-1.12.2-2.0.15.1.jar
- environmentaltech-1.12.2-2.0.15.1.jar

How to reproduce bug: 
Just create a Profile an try to build in creative.
The requirement "Structure Frame Tier x or above" keeps written in red.

**Comment by ValkyrieofNight** [comment]
No longer updating the 1.12.2 version
